### DISCUSSION TITLE
v19.21.3-4.0 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#160] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.21.3`  
Current YTLite Version: `4.0`  
Current App Compatibility: `iOS/iPadOS 14.0` or later

**RELEASE 19.21.3-4.0:** (2024-06-03)

- Updated Youtube from `19.19.5` to `19.21.3`

- Updated submodules

- Updated @daynch96 YTLite from `3.0.1` to `4.0`
    - Downloading feature
    - Tweak UI improvements
    - Added some player gestures
    - Added Import/Export settings feature (Tweak preferences)
    - Added Old/New UI toggle (Version spoofing)
    - Added options to remove community posts and inline comment under the player
    - Added OLED dark mode support
    - Added option to speed up Shorts
    - Lite version temporarily disabled
    - Fixed blank screen issue (Thanks to @PoomSmart for the https://github.com/PoomSmart/YouTube-X)
    - Removed version spoofing on extra speed options and classic video quality as unneeded
    - Fixed issues related to Disable auto-caption feature
    - Fixed crashes related to Auto-quality feature

  
### Issues in this Release:
- LowContrastMode does not work *(Compatible only with YouTube v17.33.2-v17.38.10)*

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.21.3-4.0'>v19.21.3-4.0 - YTLitePlus</a>.</em>

**Comment by S3bbz** [comment]
I can't seem to build this release. I have never experience this issue before. I am using the decrypted 19.21.3 Youtube .ipa as well. Anyone have a clue? Using catbox for hosting the youtube ipa

**Reply by cybrnook** [reply]
Switch to the tag and build instead of building from main branch. Likely something from last nights commits that aren’t compatible.

**Reply by S3bbz** [reply]
> Switch to the tag and build instead of building from main branch. Likely something from last nights commits that aren’t compatible.

I tried. There are no tags listed to build off

**Reply by abheizthe** [reply]
> > Switch to the tag and build instead of building from main branch. Likely something from last nights commits that aren’t compatible.
> 
> I tried. There are no tags listed to build off

Go to this main repo, switch to action tab and there's an artifacts on this release with compressed file

**Reply by S3bbz** [reply]
> > > Switch to the tag and build instead of building from main branch. Likely something from last nights commits that aren’t compatible.
> > 
> > 
> > I tried. There are no tags listed to build off
> 
> Go to this main repo, switch to action tab and there's an artifacts on this release with compressed file

Thank you! Got it working now

**Reply by omarawad11** [reply]
can you please upload it to google drive and provide me with a link please ? @abheizthe @S3bbz

**Reply by omarawad11** [reply]
Thank you bro @naoto0826g

**Reply by omarawad11** [reply]
@Balackburn the app crashes once i open a video , but works when i open short

**Reply by omarawad11** [reply]
@naoto0826g  does it work in your device ?

**Reply by omarawad11** [reply]
i disabled isponsorblock and it works

**Reply by abheizthe** [reply]
> i disabled isponsorblock and it works

Yesss. That's the problem. iSponsorBlock have an update yesterday, if you build before last commit of this release. It will crashed

Update the branch and rebuild it
Or, just inject/ update the latest iSponsorBlock.deb to the Yt .ipa instead of rebuild it

**Reply by abheizthe** [reply]
> > > i disabled isponsorblock and it works
> > 
> > 
> > Yesss. That's the problem. iSponsorBlock have an update yesterday, if you build before last commit of this release. It will crashed
> > Update the branch and rebuild it Or, just inject/ update the latest iSponsorBlock.deb to the Yt .ipa instead of rebuild it
> 
> Is the new version 1.2.6 correct?

Yess it is

**Reply by omarawad11** [reply]
yes, in my case i don't need isponsorBlock that much, so i can use youtube without it , thanks naoto for your help

**Reply by ghost** [reply]
> yes, in my case i don't need isponsorBlock that much, so i can use youtube without it , thanks naoto for your help

The iSposorBlock crash should be fixed now.

**Reply by omarawad11** [reply]
no ,its still occurs maybe if you give me the new file you made , it will work

**Comment by CSummerChild** [comment]
I'm not sure if I should make an issues thread about this orb if it's just like it, but the app can't seem to cast or locate other devices.

**Reply by abheizthe** [reply]
> I'm not sure if I should make an issues thread about this orb if it's just like it, but the app can't seem to cast or locate other devices.

Did you accepted "find local network"?

**Reply by CSummerChild** [reply]
Yea I did. It's just not finding any device while the normal app finds them just fine.

**Reply by abheizthe** [reply]
> Yea I did. It's just not finding any device while the normal app finds them just fine.

Sorry can't help you abiut this issue, i don't have any device to test the casting video
wait for the other's

**Comment by Jahanzeb009** [comment]
When I use YouTube 19.21.3 decrypted version it's still creating the 3.0 version, not 4.0. I have tried with the latest youtube version 19.22.6 but it is still the same.

the app is working but the latest 4.0 tweaks are not included.
